__tests__/src/index.spec.js

```diff
@@ -120,7 +120,7 @@ describe('toMatchImageSnapshot', () => {
     const customDiffConfig = { threshold: 0.3 };
     matcherAtTest('pretendthisisanimagebuffer', { customDiffConfig });
     const { diffImageToSnapshot } = require('../../src/diff-snapshot');
-    expect(diffImageToSnapshot.mock.calls[0][0].customDiffConfig).toBe(customDiffConfig);
+    expect(diffImageToSnapshot.mock.calls[0][0].customDiffConfig).toEqual(customDiffConfig);
```

Comment:
Why this change?

Reply:
Below, this is run ``Object.assign({}, commonCustomDiffConfig, customDiffConfig)``. The object is structurally equivalent (toEqual), but not reference equivalent (toBe) because the keys were shallow-copied to a new object.